tt_metal/programming_examples/eltwise_binary/eltwise_binary.cpp

```diff
@@ -138,7 +138,7 @@ int main(int argc, char **argv) {
          */
         KernelHandle eltwise_binary_kernel_id = CreateKernel(
             program,
-            "tt_metal/kernels/compute/eltwise_binary.cpp",
+            "tt_eager/tt_dnn/op_library/eltwise_binary/kernels/compute/eltwise_binary.cpp",
```

Comment:
Can probably leave these as the tt_metal one.

Reply:
sorry mistake, resolved